src/sentry/api/endpoints/group_details.py

```diff
@@ -336,26 +334,18 @@ def put(self, request, group):
                     )
                 ),
             )
-            metrics.incr(
-                "group.update.http_response",
-                sample_rate=1.0,
-                tags={"status": response.status_code, "detail": "group_details:put:response"},
-            )
             return Response(serialized, status=response.status_code)
         except client.ApiError as e:
+            logging.error(
+                "group_details:put client.ApiError",
+                exc_info=True,
+            )
             metrics.incr(
-                "group.update.http_response",
+                "workflowslo.http_response",
```

Comment:
I wanted to keep this in for now to keep monitoring, but not include it in the SLO since I am pretty certain it was being double reported.